server/blob/blob-objectstorage/src/main/java/org/apache/james/blob/objectstorage/crypto/CryptoConfig.java

```diff
@@ -0,0 +1,40 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.blob.objectstorage.crypto;
+
+import com.google.crypto.tink.subtle.Hex;
+
+public class CryptoConfig {
+    private final String salt;
+    private final String password;
```

Comment:
A good habit is to store password as `char[]`

Reply:
Interesting, can you expand on why ?

Reply:
Here are some answers:
https://stackoverflow.com/questions/8881291/why-is-char-preferred-over-string-for-passwords

the two most important are:
* prevent from logging it
* a String in memory can be kept by another process

Reply:
I am tempted to make a search throughout the codebase to verify that the array is actually used properly. 
According to the above, the only way the array really helps is if you erase it once you have used the password ... 

I'll make the change